### PR TITLE
ez_decoded---updates

### PR DESCRIPTION
**Pull request by eric-laurello**

### CODE REVIEW

File: models/silver/parser/silver__decoded_instructions.sql [hunk 1]
```diff
@@ -4,7 +4,7 @@
     materialized = 'incremental',
     incremental_predicates = ["dynamic_block_date_ranges"],
     unique_key = "decoded_instructions_id",
-    cluster_by = ['block_timestamp::DATE','_inserted_timestamp::DATE','program_id'],
+    cluster_by = ['program_id','block_timestamp::DATE','_inserted_timestamp::DATE'],
```
Comment (desmond-hui) [inline models/silver/parser/silver__decoded_instructions.sql:7]: I have some reservations about putting program_id first in terms of incremental loading performance. Can we make this a V2 table for a week and create a separate GHA workflow to run this in parallel to see performance before we make the change?

File: models/gold/core/core__ez_events_decoded.sql [hunk 2]
```diff
@@ -4,30 +4,30 @@
...
+    TRUE AS succeeded,
+    A.index,
     A.inner_index,
-    b.event_type,
-    b.program_id,
-    b.instruction,
+    A.event_type,
+    A.program_id,
+    NULL AS instruction,
     A.decoded_instruction,
```
Comment (desmond-hui) [inline models/gold/core/core__ez_events_decoded.sql:17]: I think we should split this to be

```
A.decoded_instruction:accounts::array as decoded_accounts,
A.decoded_instruction:args::variant as decoded_args,
A.decoded_instruction:error::string as decoding_error
```

**Review by desmond-hui — CHANGES_REQUESTED** [review]

File: models/gold/core/core__ez_events_decoded.sql (hunk 2 above)
Comment (eric-laurello) [reply models/gold/core/core__ez_events_decoded.sql:17]: @desmond-hui  Were you thinking about landing these so we can SO them or just breaking them out in the view is enough?

Comment (desmond-hui) [reply models/gold/core/core__ez_events_decoded.sql:17]: No SO, i think view is enough

**Review by desmond-hui — APPROVED** [review]

**Pull request merged**